apps/arena/lib/arena/game/player.ex

```diff
@@ -168,17 +169,21 @@ defmodule Arena.Game.Player do
         game_state
 
       skill ->
+        GameUpdater.broadcast_player_block_movement(game_state.game_id, player.id, skill.block_movement)
```

Comment:
I think the GameUpdater should do the broadcast, we should do the following:
```suggestion
        Process.send(self(), {:block_movement, player.id, false}, execution_duration)
```

Reply:
Sure thing! i just added a check if the skill will block the movement to send this messages since it sounds redundant to send them if not